packages/resolver/src/translators/reactjs/hooks/useResolverSetShowNotice.tsx

```diff
@@ -1,21 +1,29 @@
-import {UseMutationOptions} from '@tanstack/react-query'
+import {useState} from 'react'
 
 import {useResolver} from '../provider/ResolverProvider'
-import {useMutationWithInvalidations} from '../../../utils/useMutationWithInvalidations'
 
-export const useResolverSetShowNotice = (
-  options?: UseMutationOptions<void, Error, boolean>,
-) => {
+export const useResolverSetShowNotice = () => {
   const {showNotice} = useResolver()
+  const [isLoading, setIsLoading] = useState(false)
+  const [error, setError] = useState<Error | null>(null)
+
+  const setShowNotice = async (value: boolean) => {
```

Comment:
useMutationWithInv... can be consumed from `@yoroi/common`, update the return to expand the mutation otherwise it will be a breaking.